### DISCUSSION TITLE
DetailsListAuto DetailsRowColumn: how to text alight right justified? And can we make a Sticky Column Header?

### DISCUSSION THREAD

**Discussion by debrba** [BlazorFluentUI/BlazorFluentUI#337] [General]
Is there any attribute to set the column content right aligned.. like numeric fields while using DetailsListAuto?
Also is there any way to keep the column header sticky (always stay on top while the rows scroll) in the container div?

![DetailsListColAlign](https://user-images.githubusercontent.com/84942941/124177135-70987300-da75-11eb-8683-c16d870b6661.png)

**Comment by limefrogyank** [comment]
I think you'd have to add your own template to `ColumnItemTemplate` inside the `DetailsRowColumn` in order to do that.... 

OR you could try it via css.  It would be a little hacky.  You could try to select for the `div` that the correct `aria-colindex` attribute.

For the sticky header, that hasn't been implemented yet.  We accept pull requests!
